README.md

```diff
@@ -0,0 +1,95 @@
+<img src="Supplementary_data/dea_logo_wide.jpg" width="900" alt="Digital Earth Australia logo" />
+
+# DEA Notebooks
+
+[![DOI](https://img.shields.io/badge/DOI-10.26186/145234-0e7fbf.svg)](https://doi.org/10.26186/145234) [![Apache license](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0) [![PyPI](https://img.shields.io/pypi/v/dea-tools)](https://pypi.org/project/dea-tools/) [![Notebook testing](https://github.com/GeoscienceAustralia/dea-notebooks/actions/workflows/test_notebooks.yml/badge.svg?branch=develop)](https://github.com/GeoscienceAustralia/dea-notebooks/actions/workflows/test_notebooks.yml)
+
+<br />
+
+This is the DEA Notebooks and DEA Tools repository of [Digital Earth Australia](https://www.dea.ga.gov.au/).
+These can be viewed on the [DEA Knowledge Hub](https://docs.dea.ga.gov.au/) and used on the
+[DEA Sandbox](https://app.sandbox.dea.ga.gov.au/).
+
+The Jupyter Notebooks and Python tools in this repository are for
+analysing DEA's satellite data products. The notebooks are designed to demonstrate
+how to use the tools and data to conduct a broad range of geospatial
+analyses. They also demonstrate how to integrate with other open-source
+software such as [Open Data Cube](https://www.opendatacube.org/) and
+[xarray](http://xarray.pydata.org/en/stable/). You can see real-world
```

Comment:
"Real-world use cases" adds confusion about the "Real world examples" notebooks section. Maybe "You can view external use cases"?